.travis.yml

```
@@ -0,0 +1,10 @@
+language: node_js
+node_js:
+  - "8.15.1"
```

```suggestion
  - stable
```

This way we won't have to update the file as Node versions come and go.